browser/components/StatsContentCard/StatsContentCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './StatsContentCard.scss';
+import PropTypes from 'prop-types';
+
+
+const StatsContentCard = (props) => {
+  const { godName, imageUrl, featureStats } = props;
+
+  return (
+    <div>
+      <h1>God name: {godName}</h1>
+      <img src={imageUrl} alt="image of god" />
+      <div>
+        {Object.keys(featureStats).map(stat => <p>{stat}: {featureStats[stat]}</p>)}
```

Comment:
Aren't we passing a single stat? I'm confused.

Reply:
that's right, will update that